app-text/diffpdf/files/diffpdf-2.1.3-qt6.patch

```diff
@@ -0,0 +1,401 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+new file mode 100644
+index 0000000..c7e6d76
+--- /dev/null
++++ b/CMakeLists.txt
+@@ -0,0 +1,25 @@
++cmake_minimum_required(VERSION 3.28)
++project(diffpdf CXX)
++
++find_package(Qt6 REQUIRED COMPONENTS LinguistTools PrintSupport Widgets)
++find_package(PkgConfig REQUIRED)
++pkg_search_module(PopplerQt6 REQUIRED IMPORTED_TARGET poppler-qt6>=24.11.0)
```

Comment:
\>=24.11.0 actually required or just a Gentoo induced minimum?

Reply:
This is a Gentoo induced minimum. It does not seem reasonable to do a lot of extra work just to lower it.

Reply:
Changed it nevertheless. Tested to compile against 22.12 on Debian 12.